lib/sparse/src/index/search_context.rs

```diff
@@ -14,14 +17,16 @@ pub struct SearchContext<'a> {
     postings_iterators: Vec<IndexedPostingListIterator<'a>>,
     query: SparseVector,
     top: usize,
+    is_stopped: &'a AtomicBool,
     result_queue: FixedLengthPriorityQueue<ScoredPointOffset>, // keep the largest elements and peek smallest
 }
 
 impl<'a> SearchContext<'a> {
     pub fn new(
         query: SparseVector,
         top: usize,
-        inverted_index: &'a InvertedIndex,
+        inverted_index: &'a (impl InvertedIndex + ?Sized),
```

Comment:
```suggestion
        inverted_index: &'a impl InvertedIndex,
```
We can skip `?Sized` because it's required for tests only. We can use in tests `impl InvertedIndex` instead of `dyn InvertedIndex`